Database/Corrections/Wotlk/wotlkQuestFixes.lua

```diff
@@ -1811,9 +1862,6 @@ function QuestieWotlkQuestFixes:Load()
         [13411] = {
             [questKeys.preQuestSingle] = {10124},
         },
-        [13420] = {
```

Comment:
This removal is wrong. The quest is acquired from that object/item, but only available once friendly with Sons of Hodir faction. This requirement is already added on master, so we have a conflict here. Not sure how to make a suggestion that removes the removal.

Reply:
@Muehe / @BreakBB it's not letting me remove this. Feel free to take this out of the PR.

Reply:
Alright, will try. :ok_hand: